games/o.yaml

```
@@ -823,6 +823,25 @@
   updated: 2015-04-11
   url: http://openmohaa.sourceforge.net/
 
+- name: OpenMoM
+  type: clone
+  originals:
+  - Master of Magic
+  repo: https://github.com/Jakz/openmom
+  development: active
+  status: semi-playable
+  lang: C++
+  framework:
+  - SDL
+  license:
+  - GPL3
+  content: open
```

```suggestion
  content: commercial
```
"You need to copy all the original LBX files from the game into data/lbx to run the game."

---

I don't know why i wrote open. Sorry!